opm/simulators/linalg/bda/opencl/ChowPatelIlu.cpp

```diff
@@ -756,9 +756,7 @@ void ChowPatelIlu<block_size>::decomposition(
     for(int i = 0; i < Nb; ++i) {
         for(int k = Ut->rowPointers[i]; k < Ut->rowPointers[i+1]; ++k) {
             int j = Ut->colIndices[k];
-            if (j == i) {
-                inverter(Ut->nnzValues + k * bs * bs, invDiagVals + i * bs * bs);
-            } else {
```

Comment:
Out of curiosity (or dumbness): What exactly does this change mean (why does this result in exact inversion)?